tests/dummy/app/components/meta-form/property-group-selector.js

```diff
@@ -0,0 +1,84 @@
+import { action } from '@ember/object';
+import { guidFor } from '@ember/object/internals';
+import { tracked } from '@glimmer/tracking';
+import InputFieldComponent from '@lblod/ember-submission-form-fields/components/rdf-input-fields/input-field';
+import {
+  RDF,
+  SHACL,
+  triplesForPath,
+  updateSimpleFormValue,
+} from '@lblod/submission-form-helpers';
+import { namedNode } from 'rdflib';
+
+function byLabel(a, b) {
+  const textA = a.label.toUpperCase();
+  const textB = b.label.toUpperCase();
+  return textA < textB ? -1 : textA > textB ? 1 : 0;
+}
+
+export default class PropertyGroupSelector extends InputFieldComponent {
+  inputId = 'select-' + guidFor(this);
+
+  @tracked selected = null;
+  @tracked options = [];
+
+  constructor() {
+    super(...arguments);
+    this.loadOptions();
+    this.loadProvidedValue();
+  }
+
+  @action
+  loadOptions() {
+    const sourceGraph = this.args.graphs.sourceGraph;
+    const propertyGroup = new namedNode('http://lblod.data.gift/vocabularies/forms/PropertyGroup');
```

Comment:
I think this mixes the `namedNode` and `NamedNode` versions into one?

```suggestion
    const propertyGroup = namedNode('http://lblod.data.gift/vocabularies/forms/PropertyGroup');
```

```suggestion
    const propertyGroup = new NamedNode('http://lblod.data.gift/vocabularies/forms/PropertyGroup');
```